src/bitwarden_api.py

```diff
@@ -39,16 +39,14 @@ def _join_urls(base: str, *paths: str):
 
 
 def _get_custom_ca_certificate_location() -> Optional[str]:
-    if 'SPLUNK_HOME' not in os.environ:
-        return None
-
-    app_cacerts_file = os.path.join(os.environ.get('SPLUNK_HOME'), 'etc', 'auth',
-                                    'bitwarden_event_logs_cacerts.pem')
-    if not os.path.isfile(app_cacerts_file):
-        return None
-
-    return app_cacerts_file
-
+    path = os.environ.get(
+        "BITWARDEN_APP_CACERTS",
+        os.path.join(
+            os.environ.get("SPLUNK_HOME", ""),
+            "etc", "auth", "bitwarden_event_logs_cacerts.pem"
+        )
+    )
+    return path if os.path.isfile(path) else None
```

Comment:
I suggest that it's more explicit and safer to expressly check whether environment variables are defined, rather than providing an empty string as a default value. Example pseudo code:

- if BITWARDEN_APP_CACERTS is defined and file exists -> return that
- if SPLUNK_HOME is defined and file exists at the default path -> return that
- (else) return None

Reply:
Thanks for taking a look at this @eliykat . I updated this function based on the pseudo code you provided. Let me know if there are any other changes/updates needed.